src/components/organizations/index.jsx

```diff
@@ -0,0 +1,169 @@
+import {Box, Typography, Button} from "@mui/material";
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import ListItemText from '@mui/material/ListItemText';
+import openDataCommons from '../../Icons/svg/openDataCommons.svg';
+import drugDesign from '../../Icons/svg/drugDesign.svg';
+import NIDM from '../../Icons/svg/NIDM.svg';
+import NIF from '../../Icons/svg/NIF.svg';
+import NIDDK from '../../Icons/svg/NIDDK.svg';
+import SPARC from '../../Icons/svg/SPARC.svg';
+import RRID from '../../Icons/svg/RRID.svg';
+import {vars} from "../../theme/variables";
+import Link from "@mui/material/Link";
+import PersonAddOutlinedIcon from '@mui/icons-material/PersonAddOutlined';
+const {gray700, gray500, gray200, brand600} = vars
+
+const organizationsList = [
```

Comment:
can we simulate the fetch request in the useEffect when mounting this component, display the loader when the request is sent (a fake request for now it's fine) and when done setState for the organization list so that we can disable the loader and display the organizations? Thanks @Salam-Dalloul